### PR TITLE
Fix service

### PR DESCRIPTION
**Pull request by you-oops-dev**
You have an error in failmsg@.service.
And I still need to do the following:
sudo mkdir /etc/systemd/system/failmsg@.service.d
sudo touch /etc/systemd/system/failmsg@.service.d/toplevel-override.conf
sudo systemctl daemon-reload
A [source](https://wiki.archlinux.org/title/Systemd#Notifying_about_failed_services)

### CODE REVIEW

**Comment by dino-** [comment]
Agree about the `After=` in `[Unit]` but I'm not sure about the `WantedBy`, the docs say

```
       WantedBy=, RequiredBy=
           This option may be used more than once, or a space-separated list of unit names may be given. A symbolic link
           is created in the .wants/ or .requires/ directory of each of the listed units when this unit is installed by
           systemctl enable. This has the effect that a dependency of type Wants= or Requires= is added from the listed
           unit to the current unit. The primary result is that the current unit will be started when the listed unit is
           started. See the description of Wants= and Requires= in the [Unit] section for details.
```

This part **The primary result is that the current unit will be started when the listed unit is started** I definitely don't want. This unit should never be started unless another one fails.

I didn't know about the top-level drop-in capability, cool! I could imagine some people not liking it but I do like the idea of this default for every service's OnFailure. I'll write some explanation for this into the README in case people don't want it and add the file/dir changes.

Finally, it's my understanding that it's discouraged to do `systemctl daemon-reload` automatically on Arch Linux after installing systemd unit files. <https://bugs.archlinux.org/task/50529>

**Comment by you-oops-dev** [comment]
After reading your Readme, I realized that you need to add OnFailure=failmsg@%n.service to each Unit. As I suggested to you and did it, it turns out that everything needs to be changed at all. Well, if you partially agree with my edits, reject them. Add only those you agree with.

P.S Sorry, my English is bad, so I use a translator.

**Comment by dino-** [comment]
> After reading your Readme, I realized that you need to add OnFailure=failmsg@%n.service to each Unit. As I suggested to you and did it, it turns out that everything needs to be changed at all. Well, if you partially agree with my edits, reject them. Add only those you agree with.
> 
> P.S Sorry, my English is bad, so I use a translator.

I actually do like that suggestion, `service.d/toplevel-override.conf` instead of individually in each service. I didn't know this was available. Yes, I will use some of these things. Thank you.

**Comment by you-oops-dev** [comment]
And I thought that on the contrary you do not like it.
And I understood why you liked it due to the fact that it is automatically entered into each service file, that is, you do not need to manually enter it.
Let's make a worthy tool through mistakes and trial!

**Comment by you-oops-dev** [comment]
>I actually do like that suggestion, service.d/toplevel-override.conf instead of individually in each service. I didn't know this was available. Yes, I will use some of these things. Thank you.

It was this moment that I did not know how to reflect in the code, so I gave the link to you.

File: failmsg@.service [hunk 1]
```diff
@@ -17,7 +17,11 @@
...
 [Unit]
 Description=Failure message email for %i
+After=network.target
 
 [Service]
 Type=oneshot
 ExecStart=/usr/bin/failmsg.sh %i
+
+[Install]
+WantedBy=multi-user.target
```
Comment (dino-) [inline failmsg@.service:27]: `WantedBy` is not for this type of template unit. Please remove lines 25-27 and I will merge.

**Review by dino- — APPROVED** [review]
You know what, I'll just merge this into another branch and fix it myself. Thank you again!

**Pull request merged**